barretenberg/cpp/src/barretenberg/proof_system/op_queue/ecc_op_queue.hpp

```diff
@@ -281,6 +283,19 @@ class ECCOpQueue {
             .mul_scalar_full = 0,
         });
     }
+
+    void populate_ultra_ops(UltraOpTuple tuple)
```

Comment:
Why doesn't this update raw ops accordingly?

Reply:
Also, there is no description

Reply:
because ultra ops are populated in the circuit builder, if we'd want to populate the ultra ops and raw ops together it should be done in the ecc_op functionsthat operate on raw ops but that would be akward i think